<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STICKERMAKER</class>
 <widget class="QMainWindow" name="STICKERMAKER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>687</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="file_upload_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>загрузить изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_image_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>512</width>
      <height>512</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="top_caption_lineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bottom_caption_lineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="top_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>подпись сверху</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottom_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>подпись снизу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>скачать стикер</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="colour_select_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>185</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>выбрать цвет подписи</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_caption_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>141</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="language_switch_button">
    <property name="geometry">
     <rect>
      <x>692</x>
      <y>30</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
   </widget>
   <zorder>file_upload_button</zorder>
   <zorder>top_caption_lineEdit</zorder>
   <zorder>bottom_caption_lineEdit</zorder>
   <zorder>top_label</zorder>
   <zorder>bottom_label</zorder>
   <zorder>download_button</zorder>
   <zorder>colour_select_button</zorder>
   <zorder>display_image_label</zorder>
   <zorder>apply_caption_button</zorder>
   <zorder>clear_button</zorder>
   <zorder>language_switch_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
